src/quokka_editor_back/tests/routers/test_connection_manager.py

```diff
@@ -0,0 +1,130 @@
+import uuid
```

Comment:
If you want to test websocket itself without mocking it here is documentation that will help you with testing: https://www.starlette.io/testclient/#testing-websocket-sessions

Reply:
I wanted to test WebSocket itself in the second MR (test websockets.py)